src/app/components/tasks/TableTasks/TableTasks.tsx

```diff
@@ -23,10 +23,7 @@ const TableTasks: React.FC<Props> = ({ data, isBusy, onChange, pagination, sorti
       loading={ isBusy }
       numLoadingRows={ 10 }
       onChange={ onChange }
-      pagination={{
-        ...pagination,
-        paginationLength: 9
-      }}
+      pagination={pagination}
```

Comment:
Why did you remove the `paginationLength: 9` ? This is for a better usability of the pagination component. With a length of 9 it's possible to go 2 pages right or left instead of 1 page.